src/components/pages/home/slider/MainSlider.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable max-len */
+import { bannersData } from "../../../../mocks/featured-banners-data";
+import { useEffect, useState } from "react";
+
+//Components
+import { BtnSlider } from "../../../common/BtnSlider";
+import { Slider } from "../../../common/Slide";
+
+const banners = bannersData.results;
+
+export function MainSlider() {
+  const [slideIndex, setSlideIndex] = useState(1);
+  const [paused, setPaused] = useState(false);
+
+  const nextSlide = () => {
+    if (slideIndex !== banners.length) {
+      setSlideIndex(slideIndex + 1);
+    } else if (slideIndex === banners.length) {
+      setSlideIndex(1);
+    }
+  };
+
+  const prevSlide = () => {
+    if (slideIndex !== 1) {
+      setSlideIndex(slideIndex - 1);
+    } else if (slideIndex === 1) {
+      setSlideIndex(banners.length);
+    }
+  };
```

Comment:
same as the previous function, the ternary could be used to make the code more readable:

```suggestion
  const prevSlide = () => {
    const newIndex = slideIndex !== 1
      ? slideIndex - 1
      : banners.length;
    
    setSlideIndex(newIndex);
  };
```

Reply:
Solved